src/mesh/msh_vtkreader.cc

```diff
@@ -0,0 +1,363 @@
+/*!
+ *
+﻿ * Copyright (C) 2015 Technical University of Liberec.  All rights reserved.
+ *
+ * This program is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License version 3 as published by the
+ * Free Software Foundation. (http://www.gnu.org/licenses/gpl-3.0.en.html)
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+ *
+ *
+ * @file    msh_vtkreader.cc
+ * @brief
+ * @author  dalibor
+ */
+
+
+#include <iostream>
+#include <vector>
+#include "boost/lexical_cast.hpp"
+#include "boost/algorithm/string.hpp"
+
+#include "msh_vtkreader.hh"
+#include "msh_gmshreader.h" // TODO move exception to base class and remove
+#include "system/tokenizer.hh"
+#include "system/system.hh"
+
+#include "config.h"
+#include <zlib.h>
+
+
+/*******************************************************************
+ * Helper methods
+ */
+template<typename T>
+T read_binary_value(std::istream &data_stream)
+{
+	T val;
+	data_stream.read(reinterpret_cast<char *>(&val), sizeof(val));
```

Comment:
We need use read method for reading binary data. Method get is using for reading strings ending with new-line character (see http://www.cplusplus.com/reference/istream/istream/get/).